src/components/LessonCard/LessonCard.tsx

```diff
@@ -1,80 +1,77 @@
 import { FC } from "react";
 import styled from "styled-components";
-import shapesBgLesson1Url from "./shapes-bg-lesson1.svg";
-import shapesBgLesson2Url from "./shapes-bg-lesson2.svg";
-import shapesBgLesson3Url from "./shapes-bg-lesson3.svg";
-import shapesBgLesson4Url from "./shapes-bg-lesson4.svg";
-import shapesBgLesson5Url from "./shapes-bg-lesson5.svg";
-import shapesMascotLesson1Url from "./shapes-mascot-lesson1.svg";
-import shapesMascotLesson2Url from "./shapes-mascot-lesson2.svg";
-import shapesMascotLesson3Url from "./shapes-mascot-lesson3.svg";
-import shapesMascotLesson4Url from "./shapes-mascot-lesson4.svg";
-import shapesMascotLesson5Url from "./shapes-mascot-lesson5.svg";
+import urlLesson1MascotSvg from "./lesson-1-mascot.svg";
```

Comment:
I think as discussed with @owen26 previously, let's have these urls passed into this component (as they will be as loaded from the data source) instead of getting them baked in here.

Reply:
As this was pre-existing pattern as I also originally did it - happy for that to be done in a separate PR if you prefer.

Reply:
Hm yes, let's do it in a separate PR. :)
I actually prefer to have the lesson mascots here, since they're used only here in the LessonCard. No need for other components to know about them.
I could be persuaded otherwise though. I don't feel that strongly about it.

Reply:
Hm yes, let's do it in a separate PR. :)
I actually prefer to have the lesson mascots here, since they're used only here in the LessonCard. No need for other components to know about them.
I could be persuaded otherwise though. I don't feel that strongly about it.